package.json

```diff
@@ -3,6 +3,13 @@
   "version": "3.5.1",
   "description": "state management for Vue.js",
   "main": "dist/vuex.common.js",
+  "exports": {
+    ".": {
+      "require": "./dist/vuex.common.js",
+      "import": "./src/index.mjs"
```

Comment:
I think we should place `index.mjs` at `dist`, since `src` is not packed as NPM package. We should be able to add copying script at `scripts/build.js` to simply copy `index.mjs` from `src` dir.

Reply:
Oops, didn't take that one into account. I pushed a new commit that copies `index.mjs` into the `dist/` folder while also renaming it to `vuex.mjs` to be better in line with the other files in the folder. I have to mention that I still kept the import path like this:
```js
import Vuex from '../dist/vuex.common.js'
```
Strictly spoken this can now be reduced to
```js
import Vuex from './vuex.common.js'
```
but I chose not to do it so that `import { mapGetters } from 'vuex/index.mjs'` still works in development as well.

Reply:
Very nice! That's perfect 👍